source/includes/gcp/_ssl_certificates.md

```diff
@@ -1 +1,159 @@
-#### SSL Certificates
\ No newline at end of file
+#### SSL Certificates
+
+Represents an SSL Certificate resource.
+
+<!-------------------- LIST SSL CERTIFICATE -------------------->
+
+##### List ssl certificate
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/sslcertificates"
+
+# Example:
+```
+
+```json
+{
+  "data": [
+    {
+      "certificate": "-----BEGIN CERTIFICATE-----MIIDnDCCAoQCCQDQz2lClG3ipTANBgkqhkiG9w0BAQsFADCBjzELMAkGA1UEBhMCQ0ExDzANBgNVBAgMBlF1ZWJlYzERMA8GA1UEBwwITW9udHJlYWwxEzARBgNVBAoMCkV4YW1wbGVPcmcxFDASBgNVBAsMC0V4YW1wbGVVbml0MQ8wDQYDVQQDDAZZb2dlc2gxIDAeBgkqhkiG9w0BCQEWEWV4YW1wbGVAZ21haWwuY29tMB4XDTE5MTAwMTE3NDgxNFoXDTIwMDkzMDE3NDQBatUVaR88KgmSt8B1vPS0BRyN06jAzzwaJU79yF62qh5mdXXz4dm3mwncrL4vNxdDo+4HREsmdePwmcMT1f7f7gNRMe2OQfnntIYjkuDvuzxhBgdfqXEr2s
+      -----END CERTIFICATE-----",
+      "creationTimestamp": "2019-10-02T10:36:39.138-07:00",
+      "id": "6911678730387654321",
+      "kind": "compute#sslCertificate",
+      "name": "ssl-root-ozv",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/cmc-gcpenvironment-rjb/global/sslCertificates/ssl-root-ozv",
+      "domain": "ApiDocsDomain",
+      "organization": "ApiDocsOrg",
+      "validUntil": "2020-09-30",
+      "validFrom": "2019-10-01",
+      "issuerName": "ApiDocs",
+      "uploadDate": "2019-10-02"
+    },
+    {
+      "certificate": "-----BEGIN CERTIFICATE-----MIIDnDCCAoQCCQDQz2lClG3ipTANBgkqhkiG9w0BAQsFAELMAkGA1UEBhMCIL6BLzjFCyeJ6nMLGwC0v/ptdkT+N4XvfjRjrO1t7B8jOzZy7gqfxElXA5XLpwrWT6AASg9M5IWd87r8Cor/hCuJ25bZbHECAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAFoe0Y8qU0nTBMexn/QBatUVaR88KgmSt8B1vPS0BRyN06jAzzwaJU79yF62qh5mdXXz4dm3mwncrL4v+4HREsmdePwmcMT1f7f7gNRMe2OQfnntIYjkuDvuzxhBgdfqXEr2s
+      -----END CERTIFICATE-----",
+      "creationTimestamp": "2019-10-01T13:33:36.409-07:00",
+      "id": "2973924929495238351",
+      "kind": "compute#sslCertificate",
+      "name": "ssl-root-tcs",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/cmc-gcpenvironment-rjb/global/sslCertificates/ssl-root-tcs",
```

Comment:
replace `cmc-gcpenvironment-rjb` with `test-area`

Reply:
apply change to all selflinks.

Reply:
Made the above changes.
And also removed garbage certificate and private key body with a message in the bracket indicating what text goes here.